clases/rental.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+require_relative 'book'
+
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    person.rental << self
+    @book = book
+    book.rental << self
+  end
+end
```

Comment:
- [ ] Kindly add a method for making rentals

Reply:
Thanks so much for the review, i have a comment about this review.
the methods to create the rentals are better if I create them inside the Book and Person class because of the way that type of relations work if the most complete way to do it. 
And in the documentation before doing this project explains that the function to add rentals is not supposed to be here https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md